public/store/app/index.ts

```diff
@@ -9,13 +10,20 @@ export interface AppState {
   statusPanelState: StatusPanelState;
 }
 
+export interface SatelliteBand {
+  r: number;
+  g: number;
+  b: number;
+}
+
 export interface StatusPanelState {
   isOpen: boolean;
   contentType: StatusPanelContentType;
 }
 
 // shared data model
 export const state: AppState = {
+  currentSatelliteBand: { r: 4, g: 3, b: 2 },
```

Comment:
I just used the band number for `Landsat 8-9` for RGB. But this might be wrong based on my previous comment on the `interface SatelliteBand`.